engine/swarm/services.md

```diff
@@ -70,6 +70,130 @@ $ docker service create --name helloworld alpine ping docker.com
 9uk4639qpg7npwf3fn2aasksr
 ```
 
+### Specify the image version the service should use
+
+When you create a service without specifying any details about the version of
+the image to use, the service uses the latest version available in Docker Hub
+or your registry. You can force the service to use a specific version of
+the image in a few different ways, depending on your desired outcome.
+
+An image version can be expressed in several different ways:
+
+- If you specify a tag, the manager resolves that tag to a digest, and directs
+  workers to uses that version.
+  ```bash
+  $ docker service create --name="myservice" ubuntu:16.04
+  ```
+
+  Some tags represent discrete releases, such as `ubuntu:16.04`. Tags like this
+  will almost always resolve to a stable digest over time. It is recommended
+  that you use this kind of tag when possible.
+
+  Other types of tags, such as `latest` or `nightly`, may resolve to a new
+  digest often, depending on how often an image's author updates the tag. It is
+  not recommended to run services using a tag which is updated frequently, to
+  prevent different service replica tasks from using different image versions.
+
+- If you don't specify a version at all, by convention the image's `latest` tag
+  is resolved to aThe following two commands are equivalent:
+  ```bash
+  $ docker service create --name="myservice" ubuntu
+
+  $ docker service create --name="myservice" ubuntu:latest
+  ```
+
+- If you specify a digest directly, that exact version of the image is always
+  used.
+  ```bash
+  $ docker service create --name="myservice" ubuntu:16.04@sha256:35bc48a1ca97c3971611dc4662d08d131869daa692acb281c7e9e052924e38b1
+  ```
+
+When you create a service, the swarm manager resolves the
+image's tag to the specific digest it points to at the time of service creation.
+If the manager is able to resolve the tag to a digest, worker nodes for that
+service will use that specific digest unless the service is explicitly updated.
```

Comment:
Let's just add a line here that this is particularly important for services that use mutable tags like `latest`, so that all service tasks use the same image.